src/pages/[...slug].tsx

```diff
@@ -0,0 +1,42 @@
+import type { AtamaProps } from '@atamaco/nextjs';
+import type { MetaData } from '../lib/atama';
+import type { NextPage } from 'next';
+
+import { getStaticPropsFactory, getStaticPathsFactory } from '@atamaco/nextjs';
+import Head from 'next/head';
+import { AtamaRenderer } from '@atamaco/renderer-react';
+
+import { fetcher, COMPONENTS, LAYOUTS } from '../lib/atama';
+import { Header } from '../components/header';
+import { Footer } from '../components/footer';
+
+const ContentPage: NextPage<AtamaProps<MetaData>> = ({ data }) => (
+  <>
+    <Head>
+      <title>
+        {typeof data?.meta?.seoTitle === 'string' ? data?.meta?.seoTitle : ''}
+      </title>
+      <meta
+        name="description"
+        content={
+          typeof data?.meta?.seoDescription === 'string'
+            ? data?.meta?.seoDescription
+            : ''
+        }
+      />
+    </Head>
+    <Header />
+    <main>
+      {data && (
+        <AtamaRenderer layouts={LAYOUTS} components={COMPONENTS} data={data} />
+      )}
+    </main>
+    <Footer />
+  </>
+);
+
+export const getStaticPaths = getStaticPathsFactory(fetcher);
+export const getStaticProps = getStaticPropsFactory(fetcher);
+
+// eslint-disable-next-line import/no-default-export
+export default ContentPage;
```

Comment:
@s-atama I'm probably missing something small somewhere but eslint complains about this `default` even though it works fine on our site, any ideas?

Reply:
hm that's odd. I don't know what's going on. According to our linting configuration this should be fine:

https://github.com/AtamaCo/eslint-config-atama/blob/4f079a61a66646282cb14a53e848625102b368bc/next.js#L11-L19

Reply:
Ah, that might be it. It's looking for `pages` not `src/pages`.